pwiz_tools/examples/TextPSMReader.cpp

```diff
@@ -0,0 +1,1451 @@
+//
+// Copied from examples.cpp
+//
+//
+// Original author: Robert Burke <robert.burke@proteowizard.org>
```

Comment:
Update name and copyright.

Reply:
Caught! OK, I've removed the old verbiage. I don't know if you want me to retain some of the previous boilerplate, such as license. I've put my email at the top of all three files, at least.

Reply:
Best to keep the Apache license verbiage for consistency with the rest of the codebase.